lib/replica_pools/query_cache.rb

```diff
@@ -18,29 +18,20 @@ module QueryCache
     # select_all is trickier. it needs to use the leader
     # connection for cache logic, but ultimately pass its query
     # through to whatever connection is current.
-    def select_all(*args, **kwargs)
-      relation, name, raw_binds = args
```

Comment:
I moved these into the method signature because it made the method much easier to read to me.